Research/Research/RSDTaskResultObject.swift

```diff
@@ -99,6 +100,11 @@ public struct RSDTaskResultObject : RSDTaskRunResult, Codable {
             let nestedDecoder = try container.superDecoder(forKey: .schemaInfo)
             self.schemaInfo = try decoder.factory.decodeSchemaInfo(from: nestedDecoder)
         }
+        else if let schemaIdentifier = try container.decodeIfPresent(String.self, forKey: .schemaIdentifier),
+            let versionString = try container.decodeIfPresent(String.self, forKey: .schemaIdentifier) {
```

Comment:
So, the versionString is just a duplicate of the schemaIdentifier? Is this part of the backward compatibility?

Reply:
Yes, the "new and improved" schema is to explicitly include "assessmentIdentifier", "schemaIdentifier", and "versionString" (which going forward when we don't have to match Bridge 1.0 *could* use semantic versioning) but for reverse-compatibility I wanted to include the `RSDSchemaInfo` model serialization as well. The Kotlin-native implementation *only* uses "versionString".